spec/features/trains/admin_can_edit_train_spec.rb

```diff
@@ -43,6 +34,7 @@
     choose('train[active]', option: false)
     click_button('Update Train')
 
+    expect(page).to have_content('1 successfully updated')
```

Comment:
can we use its name instead of its id? or are we just using the the api response?